consensus/misc/eip1559.go

```diff
@@ -31,79 +29,33 @@ import (
 // - basefee check
 func VerifyEip1559Header(config *params.ChainConfig, parent, header *types.Header) error {
 	// Verify that the gas limit remains within allowed bounds
-	parentGasLimit := parent.GasLimit
-	if !config.IsLondon(parent.Number) {
-		parentGasLimit = parent.GasLimit * params.ElasticityMultiplier
-	}
-	if err := VerifyGaslimit(parentGasLimit, header.GasLimit); err != nil {
+	if err := VerifyGaslimit(parent.GasLimit, header.GasLimit); err != nil {
 		return err
 	}
 	// Verify the header is not malformed
-	if header.BaseFee == nil && config.Scroll.BaseFeeEnabled() {
-		return fmt.Errorf("header is missing baseFee")
-	}
-	// Now BaseFee can be nil, because !config.Scroll.BaseFeeEnabled()
 	if header.BaseFee == nil {
-		return nil
-	}
-	// Verify the baseFee is correct based on the parent header.
-
-	var expectedBaseFee *big.Int
-
-	// compatible check with the logic in commitNewWork
-	if config.Clique == nil || config.Scroll.BaseFeeEnabled() {
```

Comment:
this line is interesting since we use Clique. I have not checked if this would introduce some incompatibilities.

Reply:
Yes. It seems geth does not allow Clique + 1559.